fcrepo-kernel-modeshape/src/main/java/org/fcrepo/kernel/modeshape/rdf/converters/PropertyConverter.java

```diff
@@ -128,7 +129,11 @@ public static String getPropertyNameFromPredicate(final NamespaceRegistry namesp
         }
 
         final String rdfLocalname = predicate.getLocalName();
-
+        if (rdfLocalname.isEmpty()) {
+          LOGGER.info("predicate local name is empty");
+          throw new RepositoryRuntimeException(
+              "Illegal predicate local name: The localName argument may not be empty or starting with digit!");
```

Comment:
How does the test above check for an initial digit? And why would it, based on [your comment](https://github.com/fcrepo4/fcrepo4/pull/954#issuecomment-158436908)?